src/store/store.ts

```diff
@@ -245,16 +245,11 @@ export const useStore = defineStore('store', {
       return this.getBusinessInformation.startDate
     },
 
-    /** The Account Information object. */
-    getAccountInformation (): AccountInformationIF {
+    /** The Current Account object. */
+    getCurrentAccount (): AccountInformationIF {
       return this.stateModel.accountInformation
     },
 
-    /** The current account id. */
-    getAccountId (): number {
-      return this.getAccountInformation?.id || null
-    },
```

Comment:
I thought this getter wasn't used enough to warrant being a store getter, so I moved its code where it's used.